zerver/lib/actions.py

```diff
@@ -3118,8 +3113,8 @@ def do_change_subscription_property(user_profile: UserProfile, sub: Subscription
     event = dict(type="subscription",
                  op="update",
                  email=user_profile.email,
-                 property=event_property_name,
-                 value=event_value,
+                 property=database_property_name,
+                 value=database_value,
                  stream_id=stream.id,
                  name=stream.name)
     send_event(user_profile.realm, event, [user_profile.id])
```

Comment:
We can't make these changes, because we need to provide backwards-compatibility to the mobile apps.  What we can do is send an `is_muted` event in addition to the `in_home_view` event (or something), but I was imagining we'd just rewrite these events inside `server_events_dispatch.js` (with no backend changes).